src/components/board/index.js

```diff
@@ -0,0 +1,3 @@
+import Board from './Board';
```

Comment:
As you see files like this almost useless. Writing logic in index would  be better and view in another file in folder. In this case: logic in index and view in Board file.